src/Kiota.Builder/WorkspaceManagement/WorkspaceConfiguration.cs

```diff
@@ -5,12 +5,16 @@
 
 namespace Kiota.Builder.WorkspaceManagement;
 
-public class WorkspaceConfiguration : ICloneable
+public class WorkspaceConfiguration(string kiotaVersion) : ICloneable
```

Comment:
```suggestion
public class WorkspaceConfiguration(string kiotaVersion="") : ICloneable
```

Reply:
An empty string is not really a version. Why do you prefer this?

Reply:
I removed the version based on discussions with Vincent & Seb